### DISCUSSION TITLE
blog/2023/01/08/bibdesk-to-zotero-pandoc/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#54] [Blog comments]
# blog/2023/01/08/bibdesk-to-zotero-pandoc/

Tips, tricks, and rationale for converting from a single big BibTeX file to a Zotero database

https://www.andrewheiss.com/blog/2023/01/08/bibdesk-to-zotero-pandoc/

**Comment by talestomaz** [comment]
There are so many cool ideas here (like the rating and read tags!), thanks a lot for that! I'm rather a beginner in markdown, currently experimenting with Obsidian. I managed to configure the Zotero-BetterBibTex combo and I'm getting the citations rendered correctly, but I'm not getting the references at the end as you get. How did you do to get the list of references?
